src/styleToCssString.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * styleToCssString converts a react js style object to a css string value.
+ */
+
+import {isUnitlessNumber} from 'react/lib/CSSProperty';
+import hyphenateStyleName from 'fbjs/lib/hyphenateStyleName';
+
+const isArray = Array.isArray;
+const keys = Object.keys;
+
+let counter = 1;
+// Follows syntax at https://developer.mozilla.org/en-US/docs/Web/CSS/content,
+// including multiple space separated values.
+let unquotedContentValueRegex = /^(normal|none|(\b(url\([^)]*\)|chapter_counter|attr\([^)]*\)|(no-)?(open|close)-quote|inherit)((\b\s*)|$|\s+))+)$/;

```

Comment:
Wow, this looks complex and error-prone. Is there a less hacky way to do this? Is there a browser way to do this (`let el = document.createElement(); el.style.foo = 'bar'; el.getAttribute('style');`) Maybe I don't fully understand why we need to convert style to CSS string.